### ISSUE TITLE
Jobs containers: Configure image pull policy

### ISSUE THREAD

**Issue by john-haugland** [equinor/radix-platform#926]
**Is your feature request related to a problem? Please describe.**
When using an external container registry, we want to configure the pull policy for the job containers. When ghcr.io experience outage (can be hours), our user can’t run the job containers because Radix will always try to pull the container from ghcr.io even if it exists in Kubernetes with the same version already - the result is that our solution is also offline.

**Describe the solution you'd like**
We would like to configure the "imagePullPolicy" in the Radix config to "IfNotPresent" so that the production version of a job container only pull down the image if it is not available locally. Ref. https://kubernetes.io/docs/concepts/containers/images/. This will also make our system much faster when running the same image several times (avoid the pull wait-time during startup)

**Comment by adamchengtkc** [comment]
I think ifnotpresent would cause problem if we use latest tag. Because (as far as I know) Kubernetes look at the tag not the hash.

**Comment by john-haugland** [comment]
Yes, this will only work when there are version numbers in the tags. It will soon be easy to deploy containers with given version tags from a github action, ref. https://github.com/equinor/radix-platform/issues/911

**Comment by oyron** [comment]
Thank you for proposing this feature. We have discussed this and concluded that we will not implement support for this at this point. We believe that the risk for misconfiguration and unintended behaviours if we implement this will outweigh the benefits. When configuring an external container registry one will be dependant on its availability.  You could consider switching to ACR that probably have had a better availability for the past year. Closing the issue.

**Comment by john-haugland** [comment]
Maybe the best reasoning to implement this is not the github container registry outage problem, but the ability to faster launch a job container. During startup, downloading a container can easily take 15 seconds depending on the container size. Are there any possible speed improvements with a different pull policy ?
